pixi/src/checks/MobileFriendlinessCheck.js

```diff
@@ -17,6 +17,10 @@ const API_ENDPOINT = API_ENDPOINT_MOBILE_FRIENDLINESS;
 const API_URL = `${API_ENDPOINT}?key=${AMP_DEV_PIXI_APIS_KEY}`;
 
 export default class MobileFriendlinessCheck {
+  static getCheckCount() {
+    return 2;
```

Comment:
Doesn't this only render one card?

Reply:
We have the mobileFriendly and resourcesLoadable checks.